app/src/main/java/com/infomaniak/mail/utils/AttachmentMimeTypeUtils.kt

```diff
@@ -28,7 +28,7 @@ object AttachmentMimeTypeUtils {
         "application/x-pdf"
     )
 
-    private val calendarMatches = mimeTypeSetOf(
+    val calendarMatches = mimeTypeSetOf(
         "application/ics",
         "text/calendar"
```

Comment:
All last comma are missing in this file
```suggestion
        "text/calendar",
```